Docker-Swarm-deployment/single-node/immudb/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM python:3.8-slim
+
+RUN pip install immudb-py
+
+ADD script.py /home/
```

Comment:
```COPY``` is better than using ```ADD``` here. [refer](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#add-or-copy) @namanmanish